modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -102,6 +109,20 @@ namespace imgproc {
         }
     };
 
+    G_TYPED_KERNEL(GCalcOptFlowLK, <GArrOutLK(GMat,GMat,GArrayPoint2f,GArrayPoint2f,cv::Size,int,cv::TermCriteria,int,double)>,
```

Comment:
As I can see, Andrey unified these classes to avoid different macros usage: https://github.com/opencv/opencv/pull/16496. So, is it necessary to use _M modification?

Reply:
Ough, okay, you are right, there is no reason to use _M modification.